TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/PATCH-Positive-Basicinfo.robot

```diff
@@ -0,0 +1,75 @@
+
+*** Test Cases ***
+ProfilePATCH001 - Update one basicinfo of device profile
+    #Update manufacturer
+    Given Generate A Device Profile Sample  Test-Profile-1
+    And Create Device Profile ${deviceProfile}
+    And Set To Dictionary  ${deviceProfile}[0][profile]  manufacturer=Mfr_ABC
+    When Update Device Profile ${deviceProfile}
+    Then Should Return Status Code "207"
+    And Should Return Content-Type "application/json"
+    And Item Index 0 Should Contain Status Code "200"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+    And Profile Test-Profile-1 Data "manufacturer" Should Be Updated
+    And Profile Test-Profile-1 Data "model" Should Not Be Updated
+    And Profile Test-Profile-1 Data "description" Should Not Be Updated
+    And Profile Test-Profile-1 Data "label" Should Not Be Updated
+    [Teardown]  Delete Device Profile By Name  Test-Profile-1
+
+ProfilePATCH002 - Update multiple basicinfo of device profile
+    Given Generate Multiple Device Profiles Sample
+    And Create Device Profile ${deviceProfile}
+    And Set To Dictionary  ${deviceProfile}[0][profile]   model=Model_ABC
+    And Set To Dictionary  ${deviceProfile}[1][profile]   description=Dcp_ABC
+    And Set To Dictionary  ${deviceProfile}[2][profile]   label=Label_ABC
+    When Update Device Profile ${deviceProfile}
+    Then Should Return Status Code "207"
+    And Should Return Content-Type "application/json"
+    And Item Index 0 Should Contain Status Code "200"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+    And Profile Test-Profile-1 Data "model" Should Be Updated
+    And Profile Test-Profile-2 Data "description" Should Be Updated
+    And Profile Test-Profile-3 Data "label" Should Be Updated
+    [Teardown]  Delete Device Profile By Name  Test-Profile-1 Test-Profile-2 Test-Profile-3
+
+ProfilePATCH003 - Update multiple basicinfo of device profile
+    #Update manufacturer, model, description and label
+    Given Generate A Device Profile Sample  Test-Profile-1
+    And Create Device Profile ${deviceProfile}
+    And Set To Dictionary  ${deviceProfile}[0][profile]  manufacturer=Mfr_ABC
+    And Set To Dictionary  ${deviceProfile}[0][profile]   model=Model_ABC
+    And Set To Dictionary  ${deviceProfile}[0][profile]   description=Dcp_ABC
+    And Set To Dictionary  ${deviceProfile}[0][profile]   label=Label_ABC
+    When Update Device Profile ${deviceProfile}
+    Then Should Return Status Code "207"
+    And Should Return Content-Type "application/json"
+    And Item Index 0 Should Contain Status Code "200"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+    And Profile Test-Profile-1 Data "label" Should Be Updated
+    [Teardown]  Delete Device Profile By Name  Test-Profile-1
+
+*** Keywords ***
+Profile ${device_profile_name} Data "${property}" Should Be Updated
+    Query device profile by name  ${device_profile_name}
```

Comment:
This design will query the the same profile several times in the same test case.
The profile should be queried once and use the key to pass the same profile data to verify the fields.